### ISSUE TITLE
Microphtalmia

### ISSUE THREAD

**Issue by xsnajperx** [obophenotype/human-phenotype-ontology#10236]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by jagcgc** [comment]
I believe this term already exists as [Microphthalmia HP:0000568](https://hpo.jax.org/app/browse/term/HP:0000568)

**Comment by pnrobinson** [comment]
Yes, this term already exists as @jagcgc mentions. Please reopen the issue if something was missing.
